src/main/java/core/basesyntax/strategy/impl/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    private final FruitStorageDao fruitStorageDao;
+
+    public PurchaseOperationHandler(FruitStorageDao fruitStorageDao) {
+        this.fruitStorageDao = new FruitStorageDaoImpl();
+    }
```

Comment:
```suggestion
    public PurchaseOperationHandler(FruitStorageDao fruitStorageDao) {
        this.fruitStorageDao = fruitStorageDao;
    }
```

same for other handlers